api/src/iotsignals/__init__.py

```diff
@@ -4,5 +4,5 @@
 # API versions (Major, minor, patch)
 API_VERSIONS = {
     'v0': (0, 0, 2),
-    # 'v1': (1, 1, 0),
+    'v1': (1, 1, 0),
```

Comment:
Ik weet niet of deze versie ooit in gebruik was, misschien beter om hier (1, 2, 0) van te maken? Of zelfs (2, 0, 0)?